tests/test_claude_wrapper_hooks.py

```
@@ -526,23 +559,107 @@ def test_live_socket_enforces_heap_cap_for_space_separated_flag(failures: list[s
+    expect(any(" ping" in line for line in cmux_log), f"whitespace HOME fallback: expected cmux ping, got {cmux_log}", failures)
+    expect(claudecode == "__UNSET__", f"whitespace HOME fallback: expected CLAUDECODE unset, got {claudecode!r}", failures)
+    require_path = require_path_from_node_options(node_options)
+    expect(require_path != "", f"whitespace HOME fallback: expected NODE_OPTIONS restore preload, got {node_options!r}", failures)
+    expect(
+        not any(ch.isspace() for ch in require_path),
+        f"whitespace HOME fallback: expected whitespace-free restore path, got {require_path!r}",
+        failures,
+    )
+    if sys.platform == "darwin":
+        expect(
+            require_path == f"/var/tmp/cmux-{os.getuid()}/com.cmuxterm.app/cmux-claude-node-options/restore-node-options.cjs",
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/var/tmp/cmux-"

(S108)